README.md

```diff
@@ -1,19 +1,98 @@
 
-### requires.io
-[![Requirements Status](https://requires.io/github/hydroshare/hs_docker_base/requirements.svg?branch=develop)](https://requires.io/github/hydroshare/hs_docker_base/requirements/?branch=master)
-
-Hydroshare
-============
+| Workflow | Clean | Build/Deploy | Unit Tests | Flake8 | Requirements |
+| -------- | ----- | ------------ | ---------- | -------| ------------ |
+| [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-workflow/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-workflow/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-clean/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-clean/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-deploy/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-deploy/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-test/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-test/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-flake8/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-flake8/) | [![Requirements Status](https://requires.io/github/hydroshare/hs_docker_base/requirements.svg?branch=develop)](https://requires.io/github/hydroshare/hs_docker_base/requirements/?branch=master) | 
 
 HydroShare is a collaborative website being developed for better access to data and models in the hydrologic sciences. HydroShare will provide the sustainable technology infrastructure needed to address critical issues related to water quantity, quality, accessibility, and management. HydroShare will expand the data sharing capability of the CUAHSI Hydrologic Information System by broadening the classes of data accommodated, expanding capability to include the sharing of models and model components, and taking advantage of emerging social media functionality to enhance information about and collaboration around hydrologic data and models. 
 
-If you want to contribute to HydroShare, please see this document: [Getting Started with HydroShare](https://github.com/hydroshare/hydroshare/wiki/getting_started)
-
 More information can be found in our [Wiki Pages](https://github.com/hydroshare/hydroshare/wiki)
+
+## Install
+
+This README file is for developers interested in working on the Hydroshare code itself, or for developers or researchers learning about how the application works at a deeper level. If you simply want to _use_ the application, go to http://hydroshare.org and register an account.
+
+If you want to install and run the source code of application locally, read on.
+
+#### Dependencies
+[VirtualBox](https://www.virtualbox.org/wiki/Downloads) is required, as the preferred development environment is encapuslated within a VM. This VM has the appropriate version of Ubuntu  nstalled, as well as python and docker and other necessary development tools. 
```

Comment:
I suggest

### [Virtual Box](https://www.virtualbox.org/wiki/Downloads) development environment

To quickly get started developing we offer a preconfigured development environment encapsulated within a virtual box Virtual Machine (VM).  This includes the appropriate version of Ubuntu, Python, Docker, and other key dependencies and development tools.